src/vstart.sh

```diff
@@ -221,7 +221,7 @@ usage=$usage"\t--short: short object names only; necessary for ext4 dev\n"
 usage=$usage"\t--nolockdep disable lockdep\n"
 usage=$usage"\t--multimds <count> allow multimds with maximum active count\n"
 usage=$usage"\t--without-dashboard: do not run using mgr dashboard\n"
-usage=$usage"\t--bluestore-spdk <vendor>:<device>: enable SPDK and specify the PCI-ID of the NVME device\n"
+usage=$usage"\t--bluestore-spdk: enable SPDK and specify the PCI-ID(etc, 0000:81:00.0) of the NVME device with comma-separated\n"
```

Comment:
you are removing the argument spec. but we still need it for specifying the PCI-ID, right?

> with comma-separated

could you fix the syntax error? i think what you mean is "a comma-delimited list of PCI-IDs of NVME device.